src/components/preference-group/preference-group.component.html

```diff
@@ -1,7 +1,9 @@
-<label class="header" [class.enabled]="enabled" [class.always-enabled]="alwaysEnabled">
+<mat-card class="preference-group">
+  <label class="header" [class.enabled]="enabled" [class.always-enabled]="alwaysEnabled">
   <span class="title">
     {{name}}
   </span>
-  <app-switch [(value)]="value" *ngIf="!alwaysEnabled"></app-switch>
```

Comment:
You may remove `SwitchModule` and its files since it is no longer used by any other components.

Reply:
oOops, I forgot to include the removal in git commit. Thanks for pointing that out.